shipyard_airflow/control/dag_runs.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import falcon
+import requests
+
+from .base import BaseResource
+
+class DagRunResource(BaseResource):
+
+    authorized_roles = ['user']
+
+    def on_post(self, req, resp, dag_id, run_id=None, conf=None, execution_date=None):
+        req_url = 'http://localhost:32080/api/experimental/dags/{}/dag_runs'.format(dag_id)
+
+        response = requests.post(req_url,
+                                 json={
+                                 "run_id": run_id,
+                                 "conf": conf,
+                                 "execution_date": execution_date,
+                             })
+        if not response.ok:
+            raise IOError()
```

Comment:
@sh8121att as well.

Is this really what you want?

Maybe this is a falcon-ism I am unaware of.  I would assume you would want to log the response/traceback, and send a falcon.HTTP_5xx.  Optionally, the user (caller) may benefit from the response error here as well directly in their response.

We may want these applications to start defining their error codes, e.g. 

```
errors/constants/whatever.py:
REMOTE_ERROR = falcon.HTTP_503 # (making this up)
```

and here:
`resp.status = REMOTE_ERROR`